doc/guides/modules.rst

```diff
@@ -196,11 +205,12 @@ Follow the following steps to request a new module:
    created
 #. Propose a name for the repository to be created under the Zephyr project
    organization on Github.
-#. Maintainers from the Zephyr project will create the repository and initialize
-   it. You will be added as a collaborator in the new repository.
+#. If the request is approved, maintainers from the Zephyr project will create
+   the repository and initialize it. You will be added as a collaborator in the
+   new repository.
 #. Submit the module content (code) to the new repository following the
    guidelines described :ref:`here <modules_using_west>`.
-#. Add a new entry to the :zephyr_file:`west.yml` with the following
+#. Add a new entry to the `west.yml`_ manifest file with the following
```

Comment:
I'd recommend leaving this as `` :zephyr_file:`west.yml` `` (as noted below, you shouldn't lock it to master)